docs/3_troubleshooting.md

```diff
@@ -25,3 +27,5 @@ In case you encounter any issues while using ReVanced Manager, please refer to t
 The next page will teach you how to build ReVanced Manager from source.
 
 Continue: [🔨 Building from source](4_building.md)
+
+[^1]: https://old.reddit.com/r/revancedapp/comments/xna2gn/youtube_unsupported_version_and_split_apk/
```

Comment:
Here is an official Android documentation of a split architecture:

```suggestion
[^1]: https://developer.android.com/guide/app-bundle/app-bundle-format
```

Because the docs must be agnostic towards any app, I suggest against linking to a YouTube specific solution for a generalized problem. You may want us to introduce app-specific FAQs, though this would work against our design. Problems such as those mentioned in this docs should be solvable in general. If you there are issues that require specific attention, these issues should be solved somewhere else (e.g split files cause problems in many cases, so we add a warning popup if we detect its a split and warn about the possibility of patching not working)

Reply:
As an end user rather than an Android app developer, I'm not sure how I would use the information in the official Android documentation to determine the appropriate next step. That documentation doesn't say anything about how to tell whether the app your have is split or full, nor does it suggest that some apps might be available in both varieties, nor does it have any information about how one would go about finding other versions of an app, or identifying whether they are split or full once found.

I understand the desire to not associate with specific third party projects unless necessary, but applying the suggested revisions seems to me like it will result in a troubleshooting page that can't actually be used for troubleshooting by anyone except experts who didn't need help in the first place.

Reply:
This was primarily the reason it was not included in the troubleshooting page to begin with. If necessary, the footnote can have a short paragraph outlining the essence of a split APK file and a link to the documentation page for additional insight which would satisfy both, experts as well as those that seek a short and quick answer.

Reply:
Is it okay if we simply give the quick advice to get the non-`bundle` version from APKMirror which will be what 99% of users reading the page will need to do, and then link to the official documentation as a further reference?

If not, is there any wording that will both be acceptable to this project while also providing the information that users need in order to resolve their problem?

I'm just trying to reduce your support burden from being asked time and time again the same question about YouTube failing to patch, since as far as I can tell there is currently no way for a user to find out why this is happening from the documentation, or by looking at Google's reference that you linked.

I myself still don't know what advice you would give other than visiting APKMirror and downloading the non-`bundle` version of the app. Is there actually any alternative? If there is, could you be more concrete about what it is? And if there isn't any alternative, can we just document it? Neither Android nor ReVanced exist in a void; it seems somewhat counterproductive to me to try to prune out every possible reference to third party software.

Reply:
We should not mention any third-party software such as APKMirror. The user must source apps themselves and pipe them to ReVanced for patching. If there is an issue with terminology on the side of APKMirror regarding the term `bundle`, this should be solved with APKMirror. If it is not clear enough that ReVanced needs an APK file, this can be outlined by saying that a file with the extension APK is necessary.

Reply:
Okay. It sounds like the information I think users need is for some reason not allowed to be included in the documentation. I cannot say I understand the reasons given, maybe there are some legal issues I am not aware of.

In any case, I suspect that these strict policies around "acceptable content" in official documentation are the reason ReVanced gets so many repetitive support questions that waste maintainers' time, and has such bare-bones documentation in the first place. However, I acknowledge your decision.

Since the proposed improvements to the documentation have been more or less rejected, I don't think there is anything else I can do with this PR. Please feel free to take it or leave it.